cms/djangoapps/contentstore/views/assets.py

```diff
@@ -65,7 +65,7 @@ def assets_handler(request, course_key_string=None, asset_key_string=None):
         else:
             asset_key = AssetKey.from_string(asset_key_string) if asset_key_string else None
             return _update_asset(request, course_key, asset_key)
-    elif request.method == 'GET':  # assume html
+    elif request.method == 'GET' or request.method == 'POST':  # assume html
```

Comment:
Note also that when we have settled on the desired API, the documentation above must be updated.